es6-modules/src/components/todo-app.js

```diff
@@ -0,0 +1,55 @@
+import { createElement, noop } from '../lib';
+import { TodoList } from './todo-list';
+import { TodoForm } from './todo-form';
```

Comment:
`TodoList` and `TodoForm` should be a `export default`, so imported without the curly brackets.